pytest/tests/loadtest/locust/common/base.py

```diff
@@ -313,6 +314,38 @@ def is_tag_active(environment, tag):
     return run_all or opt_in or not_excluded
 
 
+def init_account_generator(parsed_options):
+    if parsed_options.shard_layout_file is not None:
+        with open(parsed_options.shard_layout_file, 'r') as f:
+            shard_layout = json.load(f)
+    elif parsed_options.shard_layout_chain_id is not None:
+        if parsed_options.shard_layout_chain_id in ['mainnet', 'testnet']:
+            shard_layout = {
+                "V1": {
+                    "fixed_shards": [],
+                    "boundary_accounts": [
+                        "aurora", "aurora-0", "kkuuue2akv_1630967379.near"
+                    ],
+                    "shards_split_map": [[0, 1, 2, 3]],
+                    "to_parent_shard_map": [0, 0, 0, 0],
+                    "version": 1
+                }
+            }
+        else:
+            sys.exit(
```

Comment:
Idea: Given that this call ends the function execution, we can improve readability by decreasing nestedness in the code:

```python
if parsed_options.shard_layout_chain_id not in ['mainnet', 'testnet']:
  sys.exit(...)

shard_layout = { ... }

```